deploy-web/package.json

```
@@ -15,8 +15,10 @@
     "postinstall": "patch-package"
   },
   "dependencies": {
+    "@akashnetwork/akash-api": "file:./akashnetwork-akash-api-0.0.0.tgz",
```

To be updated when `akash-api` is published
```suggestion
    "@akashnetwork/akash-api": "@akashnetwork/akash-api@^1.0.0",
```